frontend/src/components/activePage/ActiveLanding/ActiveLanding.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC, useState } from 'react';
+import { Col, Switch } from 'antd';
+import WorkspacesTable from '../Workspaces/WorkspacesTable';
+import TemplatesTable from '../Templates/TemplatesTable';
+import Box from '../../common/Box';
+
+export interface IWorkspace {
```

Comment:
This data should be placed in the story. for the moment pass it as a prop. Later it may become a context

Reply:
I'm not sure how to organize my data since I need its interfaces to build the `IActiveLandingProps` interface. Should I declare the `Itemplate` and `IWorkspaces` interfaces here and move the actual data structures in the story (which would then import the interfaces from this file)?

Reply:
You can export the interfaces. If they are used in a single component declare them there, instead if they are used in multiple components, it is better to declare them in a separate file (still in you working folder).
Then import the interfaces in the stories where you need them.

The point is that the component should have no hardcoded values (unless those that will be actually hardcoded when used, e.g. a particular color/setting). The data you wrote is dynamic from the perspective of the component, so if you want to use it make it a prop (or a context with the necessary decorators in the stories)